tests/accessor/accessor_implicit_conversions.h

```diff
@@ -15,6 +15,8 @@
 #include "accessor_common.h"
 
 #include <type_traits>
+// for std::function
+#include <functional>
```

Comment:
Usually we sort the `#include`

Reply:
clang-format has been applied in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/339/commits/c5d4c8e00f933f8882a5da6a765f46d77accc628